spec/controllers/books_controller_spec.rb

```diff
@@ -0,0 +1,161 @@
+require 'rails_helper'
+require_relative 'shared_context'
+
+describe BooksController do
+  it { is_expected.to route(:get, '/books').to(action: :index) }
+  it { is_expected.to route(:get, '/books/1').to(action: :show, id: 1) }
+
+  describe 'GET #index' do
+    context 'When an user is authenticated' do
+      include_context 'Authenticated User'
+      let!(:books) { create_list(:book, 5) }
```

Comment:
If you use build_stubbed instead of create you can improve your performance :wink: [Check it](https://medium.com/@DmytroVasin/speed-up-your-tests-via-build-stubbed-f1926863b3d7)

Reply:
In this case `build_stubbed_list`